src/cube_http/types/v1/meta_response.py

```diff
@@ -237,6 +237,12 @@ class V1CubeMeta(BaseModel):
         description="The unique identifier for the cube. Must be unique among all cubes and views within a deployment and follow the naming conventions."
     )
 
+    type: str | None = Field(
```

Comment:
@davidd94 can you please change this to a [literal](https://typing.python.org/en/latest/spec/literal.html) instead of string?

```py
from typing import Literal
...
type: Literal["cube", "view"] | None
...
```

This will help with Pydantic validation.

Will merge after that